BSPConvert.Lib/Source/EntityConverter.cs

```diff
@@ -209,6 +213,80 @@ private void ConvertFuncStatic(Entity funcStatic)
 			funcStatic.ClassName = "func_brush";
 		}
 
+		private void ConvertFuncPlat(Entity entity)
+		{
+			var moveDistance = 0f;
+			var brushThickness = GetBrushThickness(entity);
+
+			if (float.TryParse(entity["height"], out var height))
+				moveDistance = height + brushThickness;
+			else if (float.TryParse(entity["lip"], out var lip))
+				moveDistance = -(lip - 2 - (brushThickness * 2));
+
+			if (string.IsNullOrEmpty(entity.Name))
+			{
+				entity.Name = $"plat{platIndex}";
+				CreatePlatTrigger(entity);
+				platIndex++;
+			}
+			entity.ClassName = "func_door";
+			entity["lip"] = moveDistance.ToString();
+			entity["movedir"] = "-90 0 0";
+			entity["spawnpos"] = "1";
+			entity["spawnflags"] = "0";
+			entity["wait"] = "-1";
+		}
+
+		private void CreatePlatTrigger(Entity entity)
+		{
+			var trigger = new Entity
+			{
+				ClassName = "trigger_multiple",
+				Model = entity.Model,
+				Spawnflags = 1,
+				Origin = new Vector3(entity.Origin.X, entity.Origin.Y, entity.Origin.Z + 2)
+			};
+			trigger["parentname"] = entity.Name;
+			sourceEntities.Add(trigger);
+
+			AddPlatTriggerConnections(entity, trigger);
+		}
+
+		private void AddPlatTriggerConnections(Entity plat, Entity trigger)
+		{
+			var connection = new Entity.EntityConnection()
+			{
+				name = "onStartTouch",
+				target = plat.Name,
+				action = "close",
+				param = null,
+				delay = 0,
+				fireOnce = -1
+			};
+			trigger.connections.Add(connection);
+
+			var connection2 = new Entity.EntityConnection()
+			{
+				name = "onFullyClosed",
+				target = plat.Name,
+				action = "open",
+				param = null,
+				delay = 3,
+				fireOnce = -1
+			};
+			plat.connections.Add(connection2);
+		}
+
+		private float GetBrushThickness(Entity entity)
+		{
+			var modelIndexStr = entity["model"].Substring(1); // Removes * from model index
+			if (!int.TryParse(modelIndexStr, out var modelIndex))
+				return 0;
+			var model = q3Models[modelIndex];
```

Comment:
(nit) I typically put a new line after single line if statements. I think it helps with readability a bit and makes it clear that this code is unrelated to the if statement.

```
if (!int.TryParse(modelIndexStr, out var modelIndex))
    return 0;

var model = q3Models[modelIndex];
```